manifests/vhost.pp

```diff
@@ -362,7 +362,7 @@
   }
 
   # Load mod_alias if needed and not yet loaded
-  if ($scriptalias or $scriptaliases != []) or ($redirect_source and $redirect_dest) {
+  if ($scriptalias or $scriptaliases != []) or ($aliases or $aliases != []) or ($redirect_source and $redirect_dest) {
```

Comment:
Seems the original logic was wrong. Can you update this line to:
`if ($scriptalias and $scriptaliases != []) or ($aliases and $aliases != []) or ($redirect_source and $redirect_dest) {`